### PR TITLE
Add new Switch attribute SAI_SWITCH_ATTR_MIRROR_TC.

### PR DESCRIPTION
**Pull request by gmirek**

### CODE REVIEW

File: inc/saiswitch.h [hunk 1]
```diff
@@ -1237,6 +1237,19 @@ typedef enum _sai_switch_attr_t
     SAI_SWITCH_ATTR_FAST_API_ENABLE,
 
     /**
+     * @brief Set TC of mirrored packets
+     *
+     * This setting will apply to all mirror sessions.
+     *
+     * Default of 255 = disabled.
```
Comment (lguohan) [inline inc/saiswitch.h]: what does this disabled mean?

Comment (gmirek) [inline inc/saiswitch.h]: Added new commit with explanation of default behavior.

**Comment by kcudnik** [comment]
Don't worry about this build fail, it's fixed in #379, will work after merging it

**Comment by gmirek** [comment]
Unclear if I need to do something here, Kamil.

**Comment by gmirek** [comment]
Hi Kamil,



It is unclear to me if I need to do something with my PR. Please advise.



Thanks,

Greg



*From:* Kamil Cudnik [mailto:notifications@github.com]
*Sent:* Wednesday, January 25, 2017 5:27 PM
*To:* opencomputeproject/SAI
*Cc:* gmirek; Author
*Subject:* Re: [opencomputeproject/SAI] Add new Switch attribute
SAI_SWITCH_ATTR_MIRROR_TC. (#365)



Don't worry about this build fail, it's fixed in #379
<https://github.com/opencomputeproject/SAI/pull/379>, will work after
merging it

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/opencomputeproject/SAI/pull/365#issuecomment-275252827>,
or mute the thread

**Pull request merged**